examples/04_low_level_functions/plot_92_create_design_matrices.py

```diff
@@ -0,0 +1,42 @@
+"""
+Example of a paradigm .csv file generation: the neurospin/localizer paradigm.
+=============================================================================
+
+See Pinel et al., BMC neuroscience 2007 for reference
+"""
```

Comment:
I'm not sure whether we should keep that example or not. it boils down to creating a DataFrame to store values.
May be useful for users without any experience in Python ?